approval_frame/forms.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import unicode_literals
+
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+from registration.forms import RegistrationFormUniqueEmail
+
+class RegistrationFormCustom(RegistrationFormUniqueEmail):
+    """
+    Subclass of `RegistrationFormUniqueEmail` which includes application-
```

Comment:
(Nitpick) Line 7 indicates that `RegistrationFormCustom` is a subclass of `RegistrationFormUniqueEmail`, so we don't have to write it in the comment. Comments are only useful when they capture 'why' the code is doing something rather than 'what' the code is doing which should be evident from the code itself (by giving variables meaningful names, having a good program flow etc.). A good article on coding comments - http://blog.codinghorror.com/coding-without-comments/.